[CARBONDATA-3791] Correct spelling, link and ddl in SI and MV  Documentation (#3735)
apache/carbondata: docs/mv-guide.md

      materialized view, if not, the operation is allowed, otherwise operation will be rejected by
      throwing exception.
   3. Partition management command: `ALTER TABLE ADD/DROP PARTITION`. Note that dropping a partition

>       will be allowed only if partition is participating in all materialized views associated with fact table.

Reviewer: "if partition" can be changed to "if the partition"

Author: changed

>       will be allowed only if the partition is participating in all materialized views associated with fact table.

Reviewer 2: ```suggestion
      will be allowed only if the partition columns of fact table is participating in all of the table's materialized views.
```

Author: done

 granularity provided during creation and stored on each segment.
 
 **NOTE**:

>    1. Single select statements cannot contain time series udf(s) neither with different granularity

Reviewer 2: i think this change is not required

Author: done

   ```

 Then, the above query can be rolled up from materialized view 'agg_sales', by adding hour

>  level time series aggregation on minute level aggregation. Users can fire the EXPLAIN command

Reviewer 2: put EXPLAIN  in the quotes of code

Author: done
